tests/integration/Send-SbMessagesInBatch.tests.ps1

```diff
@@ -0,0 +1,87 @@
+[CmdletBinding()]
+param (
+    [Parameter()]
+    [PSServiceBus.Tests.Utils.ServiceBusUtils]
+    $ServiceBusUtils
+)
+
+Describe "Send-SbMessagesInBatch tests" {
+
+    # setup
+
+    # create some queues, topics and subscriptions and allow time for it to complete
+
+    $testQueue = (New-Guid).Guid
+    $ServiceBusUtils.CreateQueue($testQueue)
+
+    $testTopic = (New-Guid).Guid
+    $ServiceBusUtils.CreateTopic($testTopic)
+
+    $testSubscription = (New-Guid).Guid
+    $ServiceBusUtils.CreateSubscription($testTopic, $testSubscription)
+
+    $testMessages = @(1..2 | ForEach-Object { (New-Guid).Guid })
+
+    # send a test message to the test queue and the test topic and allow time to complete
+
+    Send-SbMessagesInBatch -NamespaceConnectionString $ServiceBusUtils.NamespaceConnectionString -QueueName $testQueue -Messages $testMessages
+    Send-SbMessagesInBatch -NamespaceConnectionString $ServiceBusUtils.NamespaceConnectionString -TopicName $testTopic -Messages $testMessages
+
+    Start-Sleep -Seconds 5
+
+    Context "Test parameter attributes" {
+
+        It "QueueName parameter should be mandatory" {
+            (Get-Command -Name Send-SbMessagesInBatch).Parameters['QueueName'].Attributes.Mandatory | Should -Be $true
+        }
+
+        It "TopicName parameter should be mandatory" {
+            (Get-Command -Name Send-SbMessagesInBatch).Parameters['TopicName'].Attributes.Mandatory | Should -Be $true
+        }
+
+        It "Message parameter should be mandatory" {
+            (Get-Command -Name Send-SbMessagesInBatch).Parameters['Messages'].Attributes.Mandatory | Should -Be $true
+        }
+    }
+
+    Context "Test sending to queue" {
+
+        It "should send a single message to the correct queue" {
+            $ServiceBusUtils.GetQueueRuntimeInfo($testQueue).MessageCountDetails.ActiveMessageCount | Should -Be 2
+        }
+
+        It "should send a message with the correct body" {
+            foreach ($testMessage in $testMessages) {
+                $ServiceBusUtils.ReceiveAndCompleteAMessage($testQueue) | Should -Be $testMessage
+            }
+        }
+    }
+
+    Context "Test sending to topic" {
+
+        It "should send a single message to the correct topic" {
```

Comment:
We just need to change the name of this test to something like "should send the correct number of messages to the correct topic"